src/pal/src/include/pal/context.h

```diff
@@ -139,6 +139,12 @@ typedef ucontext_t native_context_t;
 /////////////////////
 // Extended state
 
+// The mask for YMM registers presence flag stored in the xstate_bv. On current Linuxes, this definition is
+// only in internal headers, so we define it here. The xstate_bv is extracted from the processor xstate bit
+// vector register, so the value is OS independent.
+#ifndef XSTATE_YMM
+#define XSTATE_YMM 4
```

Comment:
Is this missing an endif?

Reply:
Right, I was deleting #endif for another bit of code and deleted this one by accident too.

Reply:
I've augmented the commit to fix this.